README.Rmd

```diff
@@ -2,24 +2,76 @@
 output: github_document
 ---
 
+<!-- README.md is generated from README.Rmd. Please edit that file -->
+
 ```{r setup, include=FALSE}
 knitr::opts_chunk$set(echo = TRUE)
 ```
 
-## flexdashboard: Easy interactive dashboards for R
+# flexdashboard <a href='https://pkgs.rstudio.com/flexdashboard'><img src='man/figures/logo.png' align="right" height="138.5" /></a>
+
+<!-- badges: start -->
+[![CRAN status](https://www.r-pkg.org/badges/version/flexdashboard)](https://CRAN.R-project.org/package=flexdashboard)
+[![R-CMD-check](https://github.com/rstudio/flexdashboard/workflows/R-CMD-check/badge.svg)](https://github.com/rstudio/flexdashboard/actions)
+<!-- badges: end -->
+
+The goal of flexdashboard is to make it easy to create interactive dashboards for R, using R Markdown.
+
+* Use [R Markdown](https://rmarkdown.rstudio.com) to publish a group of related data visualizations as a dashboard.
+
+* Support for a wide variety of components including [htmlwidgets](https://www.htmlwidgets.org); base, lattice, and grid graphics; tabular data; gauges and value boxes; and text annotations.
+
+* Flexible and easy to specify row and column-based [layouts](https://pkgs.rstudio.com/flexdashboard/reference/articles/articles/layouts.html). Components are intelligently re-sized to fill the browser and adapted for display on mobile devices.
+
+* [Storyboard](https://pkgs.rstudio.com/flexdashboard/reference/articles/articles/using.html#storyboards-1) layouts for presenting sequences of visualizations and related commentary.
+
+* Optionally use [Shiny](https://shiny.rstudio.com) to drive visualizations dynamically.
+
+* Optionally use [bslib](https://rstudio.github.io/bslib/) to easily [customize main colors and fonts](https://pkgs.rstudio.com/flexdashboard/reference/articles/articles/theme.html).
+
+Learn more about flexdashboard: <https://pkgs.rstudio.com/flexdashboard>
+
+## Examples
+
+<a href="https://beta.rstudioconnect.com/jjallaire/htmlwidgets-d3heatmap/"><img src="http://rmarkdown.rstudio.com/flexdashboard/images/htmlwidgets-d3heatmap.png" width=250 height=200></img></a>&nbsp;&nbsp;<a href="https://beta.rstudioconnect.com/jjallaire/htmlwidgets-ggplotly-geoms/"><img src="http://rmarkdown.rstudio.com/flexdashboard/images/plotly.png" width=250 height=200></img></a>&nbsp;&nbsp;<a href="https://jjallaire.shinyapps.io/shiny-biclust/"><img src="http://rmarkdown.rstudio.com/flexdashboard/images/shiny-biclust.png" width=250 height=200></img></a>
```

Comment:
Maybe worth adding `alt` descriptions, a small caption (like the one on the `examples.Rmd` page), and a link to more examples?